contributed_definitions/nyaml/NXelectronanalyser.yaml

```diff
@@ -21,10 +29,36 @@ NXelectronanalyser(NXobject):
       type: NX_CHAR
       doc: |
         Acronym or other shorthand name
-  energy_resolution(NX_FLOAT):
+  work_function(NX_FLOAT):
     unit: NX_ENERGY
     doc: |
-      Energy resolution of the electron analyser (FWHM of gaussian broadening)
+      Work function of the electron analyser.
+      
+      The work function of a uniform surface of a conductor is the minimum energy required to remove
+      an electron from the interior of the solid to a vacuum level immediately outside the solid surface.
+      
+      The kinetic energy :math:`E_K` of a photoelectron emitted from an energy-level with binding energy
+      :math:`E_B` below the Fermi level is given by :math:`E_K = h\nu - E_B - e \phi_{\mathrm{sample}}`,
+      where :math:`\phi_{\mathrm{sample}}` is the work function of the sample surface. In PES measurements,
+      the sample and the spectrometer (with work function :math:`\phi_{\mathrm{spectr.}}`) are electrically
+      connected and therefore their Fermi levels are aligned. Due to the difference in local vacuum level
+      between the sample and spectrometer, there exists an electric potential difference (contact potential)
+      :math:`\Delta\phi = \phi_{\mathrm{sample}} - \phi_{\mathrm{spectr.}}`. The measured kinetic energy of
+      a photoelectron in PES is therefore given by
+      :math:`E_K^{\mathrm{meas.}} = h\nu - E_B + \Delta \phi = h\nu - E_B - e \phi_{\mathrm{spectr.}}`.
+      As a result, the measured kinetic energy :math:`E_K^{\mathrm{meas.}}` of a photoelectron is `independent`
+      of the sample work function. Nonetheless, the work function :math:`\phi_s` needs to be known to
+      accurately determine the binding energy scale.
+  energy_resolution(NX_FLOAT):
+    unit: NX_ENERGY
+    doc:
+    - |
+      Energy resolution of the electron analyser (FWHM of gaussian broadening).
+    - |
+      xref:
+        spec: 18115-1:2023
+        term: 10.7
+        url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:10.7
   momentum_resolution(NX_FLOAT):
```

Comment:
Should we not also use NXresolution here? Do we also have ISO definitions for angular, momentum resolution?

Reply:
See #124